Add README to all charts (#35135)
istio/istio: manifests/charts/base/README.md

# Istio base Helm Chart

> This chart installs the Istio base configurations. This includes Istio CRDs.

Reviewer: ```suggestion
This chart installs resources shared by all Istio revisions. This includes Istio CRDs.
```
